dev-server/scripts/v2/migrations/20191226175528_import_abilities.js

```diff
@@ -0,0 +1,47 @@
+const path = require("path");
+const {
+  getDirectoryContent,
+  findEntityByLanguageName
+} = require("../migrations_utils");
+
+const ABILITY_DIR = path.join(__dirname, "../../../data/ability");
+
+const createAbilityTable = async knex =>
+  knex.schema.createTable("ability", table => {
+    table
+      .integer("id")
+      .unsigned()
+      .notNullable()
+      .primary();
+    table.string("name_en").notNullable();
+    table.string("name_fr");
```

Comment:
They should all have a French name.

```suggestion
    table.string("name_fr").notNullable();
```

Reply:
I'm not sure we have all values for this :/